tools/depends/target/curl/CURL-VERSION

```diff
@@ -0,0 +1,8 @@
+LIBNAME=curl
+VERSION=8.1.2
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.bz2
```

Comment:
Ive just uploaded the tarball, however ive gone with the tar.xz (better compression). It'll probably take a couple hours for mirrors to do their job fully

When you drop the base_url commit, can you update this to 
```
ARCHIVE=$(LIBNAME)-$(VERSION).tar.xz
```
Drop the SOURCE variable, as we dont have a purpose for it as far as im aware anymore.